docs/tutorials/OIDC-Based-Activation/Readme.md

```diff
@@ -0,0 +1,66 @@
+# OIDC Based Activation
+
+<!-- AUTHOR marekstransky 2024-12-16T00:00:00Z -->
+<!-- SIDEBAR _Sidebar.md sticky -->
+<!-- TEMPLATE tutorial -->
+<!-- COVER_IMAGE ThirdParty_OIDC_Login.webp -->
+
+OpenID Connect ([OIDC](https://openid.net/developers/how-connect-works/)) has become a leading interoperable authentication protocol built on [OAuth 2.0](https://oauth.net/2/), the industry-standard protocol for authorization.
```

Comment:
The whole introduction test is mainly copy&paste from the post from Grace https://www.wultra.com/blog/wultra-now-supports-oidc----heres-what-it-means-for-secure-identity-verification